pkg/datastore/postgres/postgres.go

```diff
@@ -48,12 +48,8 @@ func (db *DB) CreateNode(node *core.Node) error {
 	err := db.QueryRow(
 		`INSERT INTO eth_nodes (genesis_block, network_id, eth_node_id, client_name)
                 VALUES ($1, $2, $3, $4)
-                ON CONFLICT (genesis_block, network_id, eth_node_id)
-                  DO UPDATE
```

Comment:
Trying to understand the implications of this - but how come we're changing what we're updating on conflict? Also, kind of wondering if we want to do anything on conflict. 🤔

Reply:
Yeah sorry, this bit is confusing. You're totally right, we don't want to do anything on conflict-- and this 'update' will never really do anything, since the conflict will only occur when the `genesis_block`s are already equal.

Unfortunately, Postgres won't return anything from a `RETURNING` statement (line 57) after an `ON CONFLICT DO NOTHING`. So this is just an arbitrary, pointless update that's only there to get line 57 to work.

Reply:
Thinking I'll add a comment like "pointless update to make RETURNING work" or something like that

Reply:
Can we do a get or create? Similar to: https://github.com/makerdao/vulcanizedb/blob/staging/libraries/shared/repository/address_repository.go#L39

Reply:
Wondering whether answer #2 from [this](https://stackoverflow.com/questions/34708509/how-to-use-returning-with-on-conflict-in-postgresql) is relevant here

Reply:
Yeah good call. That version seems a little more readable